modules/net-vpc/subnets.tf

```diff
@@ -89,6 +89,14 @@ locals {
     for subnet in var.subnets_l7ilb :
     "${subnet.region}/${subnet.name}" => subnet
   }
+  subnets_l7rlb = {
```

Comment:
I think we should add an example of the net-vpc module using these 2 subnets ranges in the README.md file.

Reply:
done